### PR TITLE
Add decode methods

### PR DESCRIPTION
**Pull request by ukarlsson**

### CODE REVIEW

File: src/utils/updates.js [hunk 1]
```diff
@@ -112,6 +112,31 @@ export const logUpdateV2 = (update, YDecoder = UpdateDecoderV2) => {
...
+export const decodeUpdateV2 = (update, YDecoder = UpdateDecoderV2) => {
+  const structs = []
+  const updateDecoder = new UpdateDecoderV1(decoding.createDecoder(update))
+  const lazyDecoder = new LazyStructReader(updateDecoder, false)
+  for (let curr = lazyDecoder.curr; curr !== null; curr = lazyDecoder.next()) {
+    structs.push(curr)
+    curr.info
+  }
+  return {
+    'Structs': structs,
```
Comment (dmonad) [inline src/utils/updates.js:135]: Can you please rename this to `'structs'` and `ds`.  Then I'd be happy to get this merged.

**Review by dmonad — CHANGES_REQUESTED** [review]

File: src/utils/updates.js (hunk 1 above)
Comment (ukarlsson) [reply src/utils/updates.js]: Thanks, I have addressed the comment!

File: src/utils/updates.js [hunk 2]
```diff
@@ -112,6 +112,31 @@ export const logUpdateV2 = (update, YDecoder = UpdateDecoderV2) => {
...
+ * @param {typeof UpdateDecoderV2 | typeof UpdateDecoderV1} [YDecoder]
+ *
+ */
+export const decodeUpdateV2 = (update, YDecoder = UpdateDecoderV2) => {
+  const structs = []
+  const updateDecoder = new UpdateDecoderV1(decoding.createDecoder(update))
+  const lazyDecoder = new LazyStructReader(updateDecoder, false)
+  for (let curr = lazyDecoder.curr; curr !== null; curr = lazyDecoder.next()) {
+    structs.push(curr)
+    curr.info
```
Comment (dmonad) [inline src/utils/updates.js:132]: What is this for?

Comment (ukarlsson) [reply src/utils/updates.js]: Oh sorry that was added by mistake. Fixed

**Comment by dmonad** [comment]
Thank you!

**Pull request merged**